packages/module/patternfly-docs/content/extensions/component-groups/examples/LogSnippet/LogSnippetExample.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import LogSnippet from '@patternfly/react-component-groups/dist/dynamic/LogSnippet';
+
+export const BasicExample: React.FunctionComponent = () => {
+  const code = `apiVersion: helm.openshift.io/v1beta1/
+  kind: HelmChartRepository
+  metadata:
+  name: azure-sample-repo
+  spec:
+  connectionConfig:
+  url: https://raw.githubusercontent.com/Azure-Samples/helm-charts/master/docs`;
+
+  return <>
+    <LogSnippet message='A log snippet' logSnippet={code} />
```

Comment:
We could maybe adjust the label to make it more explicit what is it's purpose
```suggestion
    <LogSnippet message='Failure - check logs for details' logSnippet={code} />
```